src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -1225,10 +1227,12 @@ public interface IHostUISupportsMultipleChoiceSelection
         /// implementation.
         /// </returns>
         /// <seealso cref="System.Management.Automation.Host.PSHostUserInterface.PromptForChoice"/>
-        Collection<int> PromptForChoice(string caption, string message,
-            Collection<ChoiceDescription> choices, IEnumerable<int> defaultChoices);
+        Collection<int> PromptForChoice(string? caption, string? message,
```

Comment:
Please add in XML comment - implementations should never return null.

Reply:
This will be a compiler warning - I don't see the point of stating the obvious.